### DISCUSSION TITLE
Is a webhook the right way? And if, how?

### DISCUSSION THREAD

**Discussion by webwurm** [AntlerVC/firetable#1013] [Support / Q&A] [answered]
I have a question if webhooks are a good method in this usecase:

I have a firestore-collection "messages" with the fields "user_ref" (in which the reference to the user a message receives is stored) and "viewed" (True/False, if msg has been read).

Now I want to implement a button in my GUI "Mark all messages as read". So I want to cycle through the messages-collection, filtered by this user, and set all the "viewed" to True.

Is this a task for a webhook? How do I start implementing it in Rowy? The docs don't give me too much information. I'd be happy if somebody can point me in the right direction! Thanks a lot!

**Comment by bofeiw** [comment, marked as answer]
You can use the Firestore sdk like [@firebase/firestore npm package](https://www.npmjs.com/package/@firebase/firestore) in your app to read/write values to your Firebase. 

We have Webhooks feature but it is not suitable for your use case. It is intended for receiving information from third parties. However we are planning to work on proper APIs (get/post requests) in Rowy.
